Makefile

```diff
@@ -100,41 +142,35 @@ _archive: clean build-ios build-tvos
 	-framework $(IOS_ARCHIVE_PATH)$(PROJECT_NAME).framework -debug-symbols $(IOS_ARCHIVE_DSYM_PATH)$(PROJECT_NAME).framework.dSYM \
 	-framework $(TVOS_ARCHIVE_PATH)$(PROJECT_NAME).framework -debug-symbols $(TVOS_ARCHIVE_DSYM_PATH)$(PROJECT_NAME).framework.dSYM -output ./build/$(PROJECT_NAME).xcframework
 
-test: test-ios test-tvos
+test: unit-test-ios functional-test-ios unit-test-tvos functional-test-tvos
 
-test-ios: clean-ios-test-files
+unit-test-ios:
 	@echo "######################################################################"
-	@echo "### Testing iOS"
+	@echo "### Unit Testing iOS"
```

Comment:
nit: do we want to add an @echo line below as the other repos?
Same for the other echo below.